### ISSUE TITLE
Processing behind proxy has a problem with references (apparently)

### ISSUE THREAD

**Issue by mrkafk** [kermitt2/grobid#293]
1.2.3.4 - - [08/Mar/2018:08:21:35 +0000] "POST /api/processFulltextDocument HTTP/1.1" 200 12433 "http://2.3.4.5:8070/" "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.84 Safari/537.36" 31167
	INFO  [2018-03-08 08:21:37,728] org.apache.http.impl.execchain.RetryExec: I/O exception (java.net.NoRouteToHostException) caught when processing request to {s}->https://api.crossref.org:443: No route to host (Host unreachable)
	INFO  [2018-03-08 08:21:37,729] org.apache.http.impl.execchain.RetryExec: Retrying request to {s}->https://api.crossref.org:443
	INFO  [2018-03-08 08:21:40,727] org.apache.http.impl.execchain.RetryExec: I/O exception (java.net.NoRouteToHostException) caught when processing request to {s}->https://api.crossref.org:443: No route to host (Host unreachable)
	INFO  [2018-03-08 08:21:40,727] org.apache.http.impl.execchain.RetryExec: Retrying request to {s}->https://api.crossref.org:443
	INFO  [2018-03-08 08:21:43,728] org.apache.http.impl.execchain.RetryExec: I/O exception (java.net.NoRouteToHostException) caught when processing request to {s}->https://api.crossref.org:443: No route to host (Host unreachable)
	INFO  [2018-03-08 08:21:43,728] org.apache.http.impl.execchain.RetryExec: Retrying request to {s}->https://api.crossref.org:443
	INFO  [2018-03-08 08:21:46,731] org.grobid.core.utilities.Consolidation: ERROR (-1) : java.net.NoRouteToHostException thrown during request execution : works/ (,query.title=My PDF Document,rows=5,query.author=Rampal)

I have set relevant proxy info in `gradle.properties`, both in `~/.gradle` and in project directory:

```
% cat gradle.properties
#Thu, 21 Apr 2016 18:39:55 +0200
version=0.6.0-SNAPSHOT
# Set workers to 1 that even for parallel builds it works. (I guess the shadow plugin makes some trouble)
org.gradle.workers.max=1



systemProp.http.proxyHost=myhost.com
systemProp.http.proxyPort=8080
systemProp.http.proxyUser=myuser
systemProp.http.proxyPassword=mypass
systemProp.http.nonProxyHosts=*.nonproxyrepos.com|localhos

systemProp.https.proxyHost=myhost.com
systemProp.https.proxyPort=8080
systemProp.https.proxyUser=myuser
systemProp.https.proxyPassword=mypass
systemProp.https.nonProxyHosts=*.nonproxyrepos.com|localhos

```

I submitted the PDF via web interface, `TEI` | `Process Fulltext Document`.

Environment:
```
 % gradle --version

------------------------------------------------------------
Gradle 4.6
------------------------------------------------------------

Build time:   2018-02-28 13:36:36 UTC
Revision:     8fa6ce7945b640e6168488e4417f9bb96e4ab46c

Groovy:       2.4.12
Ant:          Apache Ant(TM) version 1.9.9 compiled on February 2 2017
JVM:          1.8.0_162 (Oracle Corporation 25.162-b12)
OS:           Linux 2.6.32-696.el6.x86_64 amd64
```

**Comment by kermitt2** [comment]
Thanks for reporting! 
It was not tested since a while ;)

**Comment by kermitt2** [comment]
Hello @mrkafk !

I've forgotten to report that it was fixed with commit d58c820291df7a28b6bd92e067aefab03f4720b2 in late November.

CrossRef API is now working fine under a Proxy (tested behind a CNRS proxy).

**Comment by kermitt2** [comment]
So as this is fixed and is working well with my tests, I am closing the issue. Feel free to re-open it if you have any problems.
